tools/firtool/firtool.cpp

```diff
@@ -65,6 +65,10 @@ static cl::opt<std::string>
                    cl::desc("Output filename, or directory for split output"),
                    cl::value_desc("filename"), cl::init("-"));
 
+static cl::opt<bool>
```

Comment:
This looks fine as a separate patch.  Plz add a testcase

Reply:
Factored out into #1603.